spec/facades/food_facade_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'FoodFacade' do
+  it "returns an array of food objects" do
+    stub_food_request
```

Comment:
Love that you used a helper method and stubbed the requrest. However, I would want this testing to be a bit more thorough, which could be accomplished with sad paths and breaking up your facade into 2 methods.